lms/djangoapps/grades/tasks.py

```diff
@@ -91,9 +90,6 @@ def _recalculate_subsection_grade(self, **kwargs):
     """
     try:
         course_key = CourseLocator.from_string(kwargs['course_id'])
-        if not PersistentGradesEnabledFlag.feature_enabled(course_key):
-            return
```

Comment:
Let's keep an eye on task execution times after this change goes live. I'd expect it to rise, but not terribly so. FYI @jibsheet